src/components/viewer/Viewer.tsx

```diff
@@ -44,10 +58,9 @@ export default class Viewer extends React.Component<Partial<IViewerProps>, {}> {
       <Widget
         className={classNames('pp-ui', styles.self)}
         visible={this.props.visible}
-        invertedX={this.props.invertedX}
-        invertedY={this.props.invertedY}
         locationX={this.props.locationX}
         locationY={this.props.locationY}
+        calculateInverted={true}
```

Comment:
It should be possible to just write `calculateInverted` here, unless TS has a problem with this 😛

Reply:
Actually it does :D or at least it had in some of our early default configurations